### ISSUE TITLE
Annotation of dates and times

### ISSUE THREAD

**Issue by rhdunn** [UniversalDependencies/UD_English-GENTLE#3]
When validating the `NUM` form text based on the assigned `NumForm`, I've found the following entries expressing dates and times:
```
ERROR: Sentence GENTLE_legal_service-60 -- unknown NUM NumForm=Word form '08/19/2019'
ERROR: Sentence GENTLE_syllabus_opensource-19 -- unknown NUM NumForm=Word form '3:35'
ERROR: Sentence GENTLE_syllabus_opensource-19 -- unknown NUM NumForm=Word form '5:05'
ERROR: Sentence GENTLE_syllabus_opensource-19 -- unknown NUM NumForm=Word form '10:00'
ERROR: Sentence GENTLE_syllabus_opensource-19 -- unknown NUM NumForm=Word form '11:30'
ERROR: Sentence GENTLE_syllabus_opensource-38 -- unknown NUM NumForm=Word form '10:30'
ERROR: Sentence GENTLE_syllabus_opensource-38 -- unknown NUM NumForm=Word form '12:35'
ERROR: Sentence GENTLE_threat_bolin-1 -- unknown NUM NumForm=Word form '6/25/90'
```

My understanding is that `NumForm=Word` should be used for words only.

The `NumForm=Combi` feature looks to be the best current fit for these, but the documentation implies that these are for ordinals like `1st`.

The best approach here would seem to be to add a `NumForm=Date` for `08/19/2019`, etc. and `NumForm=Time` for `10:30`, etc. This best describes the number form.

**Comment by rhdunn** [comment]
Note that EWT uses `NumForm=Digit` for these, so that should be used to be consistent with the current annotations.

**Comment by amir-zeldes** [comment]
I wouldn't mind having a `Date` value, though then we'd need `Time` too, right? I see @dan-zeman approves of `Digit` so I can go with that for now, but if someone wants Date/Time as values let me know.

**Comment by rhdunn** [comment]
It looks like https://universaldependencies.org/u/feat/NumType.html is the best place for a `Date` and `Time` value. E.g.:
- `08/19/2019` -- `NumType=Date|NumForm=Digit`
- `10:00` -- `NumType=Time|NumForm=Digit`

**Comment by amir-zeldes** [comment]
Sure, I could go with either of these for GU English datasets. Thoughts @dan-zeman @nschneid ?

**Comment by nschneid** [comment]
I dunno, it starts to feel like an entity annotation. Do we start having to add it to all numbers in dates ("Dec. 4", "the 4th")?  If we say it's because dates are orthographically special, what about URLs and other such entities with special written conventions?

My impression is that `NumForm` is meant to be focused on orthography. So I'd be fine with a shallower and crisper definition along the lines of "if it contains numeric characters and no alphabetic characters, it's `NumForm=Digit`". Or if we were to add a value, something like `Mixed` for digits + nonalphanumeric characters.

**Comment by rhdunn** [comment]
I agree that `NumForm` isn't the right place, as that is defining the lexical space (number, word, mixed, roman), whereas `NumType` is the right place as it has more semantic/structural labelling:
- `NumType=Card` -- cardinals: 1234 ; 1,234 ; one ; IXVII ; etc.
- `NumType=Ord` -- ordinals: 1st ; 45th ; sixth ; etc. [1]
- `NumType=Mult` -- double ; triple ; etc.
- `NumType=Frac` -- two-fifths ; 3/4 ; 1.25 ; etc.
- etc.

This is useful for me implementing a form text validator, as I can be stricter on what characters are allowed for these form texts. -- This can pick up inconsistencies where GUM uses `NumType=Frac` for `1.25` but EWT uses `Card`. -- It can also e.g. restrict `/` to e.g. `NumType=Frac,Date`.

If the parts are separate tokens (e.g. `10`, `:`, `30`, `am`), there don't need to be separate `NumType` values for these and would be easier to integrate with other processes, as downstream consumers won't then need to retokenize the parts.

I don't mind either approach, just that there is a consistent approach across the treebanks.

**Comment by nschneid** [comment]
See also UniversalDependencies/UD_English-EWT#337 regarding fractions.

With tokenization we try to respect established standards within the language. While fractions may not have an established standard, I would not expect 10:30 as a time to be split into "10 : 30".

**Comment by amir-zeldes** [comment]
OK, I guess we'll go with digit for now then!
